sut/g292_z43_q18/.cm/meta.json

```diff
@@ -0,0 +1,37 @@
+{
+  "data": {
+    "accelerator_frequency": "",
+    "accelerator_host_interconnect": "16x PCIe 8 lane Gen4",
+    "accelerator_interconnect": "",
+    "accelerator_interconnect_topology": "",
+    "accelerator_memory_capacity": "16 GB",
+    "accelerator_memory_configuration": "LPDDR4x",
+    "accelerator_model_name": "QUALCOMM Cloud AI 100 PCIe/HHHL Pro",
+    "accelerator_on-chip_memories": "144 MB",
+    "accelerators_per_node": "16",
+    "cooling": "air",
+    "division": "closed",
+    "framework": "Qualcomm Cloud AI SDK v1.6.80",
+    "host_memory_capacity": "16x 64 GB",
+    "host_memory_configuration": "",
+    "host_networking": "",
+    "host_networking_topology": "",
+    "host_processor_caches": "L1d: 32K; L1i: 32K; L2: 512K; L3: 256M",
+    "host_processor_core_count": "64",
+    "host_processor_frequency": "2000 MHz",
+    "host_processor_interconnect": "1x 16-20GT/s xGMI2 (?)",
+    "host_processor_model_name": "AMD EPYC 7713 64-Core Processor",
+    "host_processors_per_node": "2",
+    "host_storage_capacity": "1 TB",
+    "host_storage_type": "1x SSD",
+    "hw_notes": "With 75W Accelerator TDP constraints",
+    "number_of_nodes": "1",
+    "operating_system": "CentOS 7 (Linux kernel: 5.4.1-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:21:13 EST 2019 x86_64 x86_64 x86_64 GNU/Linux)",
+    "other_software_stack": "Collective Knowledge v2.6.1",
```

Comment:
https://github.com/mlcommons/inference_results_v2.0/blob/master/closed/Qualcomm/systems/r282_z93_q8e-qaic-v1.6.80-aic100.json#L29
```
  "other_software_stack": "Docker v20.10.12; Python v3.8.12; GCC v11.2.1; Collective Knowledge v2.6.1",
```
(also in other SUT entries)